systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -304,6 +362,22 @@ void testPushIntoImageStream() {
     }
 
     @ParallelTest
+    @TestDoc(
+        description = @Desc("Test updating and validating Kafka Connect with another plugin."),
+        steps = {
+            @Step(value = "Create TestStorage instance", expected = "Instance of TestStorage is created"),
+            @Step(value = "Generate random topic name and create Kafka topic", expected = "Kafka topic is successfully created"),
+            @Step(value = "Deploy Kafka Connect and Scraper pod with specific configurations", expected = "Kafka Connect and Scraper pod are successfully deployed"),
+            @Step(value = "Deploy network policies for KafkaConnect", expected = "Network policies are successfully deployed"),
+            @Step(value = "Create and validate EchoSink KafkaConnector", expected = "EchoSink KafkaConnector is successfully created and validated"),
```

Comment:
What does "validate" here means? How is it validated?

Reply:
That is already written in the `expected` scope so I will remove this.